src/components/charts/other.js

```diff
@@ -0,0 +1,264 @@
+    this.setState({ rerenderChart: true }, () => {
+      this.setState({ rerenderChart: false })
+    })
+  }
+
+  updateBoxplotConfiguration() {
+    this.setState({ rerenderChart: true }, () => {
+      this.setState({ rerenderChart: false })
+    })
+  }
+
+  updateGaugeConfiguration() {
+    this.setState({ rerenderChart: true }, () => {
+      this.setState({ rerenderChart: false })
+    })
+  }
+
+  updatePyramidConfiguration() {
+    this.setState({ rerenderChart: true }, () => {
+      this.setState({ rerenderChart: false })
+    })
+  }
+
+  updateWordcloudConfiguration() {
+    this.setState({ rerenderChart: true }, () => {
+      this.setState({ rerenderChart: false })
+    })
+  }
+
+  updateSankeyConfiguration() {
+    this.setState({ rerenderChart: true }, () => {
+      this.setState({ rerenderChart: false })
+    })
+  }
+
+  dropdownClickHandler(input) {
+    const mode = input.target.innerHTML;
+    const { configurations } = this.state;
+    switch (mode) {
+      case modes.heatmap: {
+        this.initHeatmap();
+        break;
+      }
+      case modes.tilemap: {
+        this.initTilemap();
+        break;
+      }
+      case modes.polar: {
+        this.initPolar();
+        break;
+      }
+      case modes.boxplot: {
+        this.initBoxplot();
+        break;
+      }
+      case modes.gauge: {
+        this.initGauge();
+        break;
+      }
+      case modes.pyramid: {
+        this.initPyramid();
+        break;
+      }
+      case modes.wordcloud: {
+        this.initWordcloud();
+        break;
+      }
+      case modes.sankey: {
+        this.initSankey();
+        break;
+      }
+      default: {
+        console.log("This mode is not implemented yet");
+      }
+    }
+    this.setState({ currentMode: mode, configurations });
+  }
+
+  renderOptionsDropdown() {
+    return (
+      <div className="dropdown">
+        <button className="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">Configurations
+        <span className="caret"></span></button>
+        <ul className="dropdown-menu">
+          <li className="dropdown-header">Other Charts</li>
+          <li><a onClick={this.dropdownClickHandler}>{modes.heatmap}</a></li>
+          <li><a onClick={this.dropdownClickHandler}>{modes.tilemap}</a></li>
+          <li><a onClick={this.dropdownClickHandler}>{modes.polar}</a></li>
+          <li><a onClick={this.dropdownClickHandler}>{modes.boxplot}</a></li>
+          <li><a onClick={this.dropdownClickHandler}>{modes.gauge}</a></li>
+          <li><a onClick={this.dropdownClickHandler}>{modes.pyramid}</a></li>
+          <li><a onClick={this.dropdownClickHandler}>{modes.wordcloud}</a></li>
+          <li><a onClick={this.dropdownClickHandler}>{modes.sankey}</a></li>
+        </ul>
+      </div>
+    )
+  }
+
+  renderConfigurationsArea() {
+    const { currentMode } = this.state;
+    switch (currentMode) {
+      case modes.heatmap: {
+        return <div> HEATMAP </div>;
```

Comment:
Here is no need for spaces between tags.

Reply:
Yes, I know, but these ones were temporary caps to test if everything works correctly... This section will be changed soon by configurations options...